interface/backend/auth.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.auth.models import User
+
+class VmckAuthBackend:
```

Comment:
Are you sure you need your own authentication backend? We're going to use LDAP anyway, so this code will likely go away. Why not simply use `./manage.py createsuperuser` for now?